python/neuroglancer/ingest/base.py

```diff
@@ -1,51 +1,72 @@
 import string
-import random
 import os
 import shutil
+import tempfile
 import subprocess
 from google.cloud import storage
 
-BUCKET_NAME = 'neuroglancer'
-QUEUE_NAME = 'pull-queue'
-PROJECT_NAME = 'neuromancer-seung-import'
+import lib
```

Comment:
I wanted to see what you had to say before messing too much with your code. I'm happy to merge them.